etc/validation-error-message-formatter.js

```diff
@@ -1,401 +1,4 @@
-// ATTENTION: Please keep these entries organized in alphabetical order
```

Comment:
As I mentioned in the pull request description, the contents of this file have been moved verbatim to `src/validation-error-message-formatter.js`.